src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -60,6 +70,16 @@ static int execute_app(
     const int argc,
     const pal::char_t* argv[])
```

Comment:
Currently (both in the existing code and with my change) it is a first one wins situation, where the first one that gets there loads hostpolicy and then subsequent calls just reuse that same hostpolicy (even if it was resolved to a different path..). We could do something with tracking the loaded hostpolicy path and checking that future requests to load actually want the same one instead of just going ahead and using the same one.